plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/file/FileHiveMetastore.java

```diff
@@ -311,9 +311,6 @@ else if (table.getTableType().equals(EXTERNAL_TABLE.name())) {
                 throw new TrinoException(HIVE_METASTORE_ERROR, "Could not validate external location", e);
             }
         }
-        else {
```

Comment:
why it's unnecessary? I think it defends us against unexpected table types in the future